### PR TITLE
Fix the default argument of decorator

### PR DESCRIPTION
**Pull request by marimeireles**
@SylvainCorlay can you merge this? Just a typo.

### CODE REVIEW

**Comment by marimeireles** [comment]
But an important one, that deserves a releas.e

**Comment by serge-sans-paille** [comment]
Does that mean we should somehow test that?

**Comment by marimeireles** [comment]
@serge-sans-paille You're right. We should do this. 
Will add a test.

**Comment by marimeireles** [comment]
@serge-sans-paille we'll need a new version after that. Can you do it? Or should I open a PR for that?
Thanks.

**Comment by marimeireles** [comment]
As soon as we merge this and create a new version we can release the blog post.

File: tests/test_basic_decorator_deprecated.py [hunk 1]
```diff
@@ -55,3 +55,20 @@ def foo(): pass
...
+
+
+    def test_default_kwarg(self):
+        code = '''
+            import deprecated
+
+            @deprecated.deprecated(reason='use another function')
+            def foo(): pass
+
+            foo()'''
```
Comment (serge-sans-paille) [inline tests/test_basic_decorator_deprecated.py:74]: I fear that test case is incomplete: ` checkDeprecatedUses` is never called :-)

Comment (serge-sans-paille) [reply tests/test_basic_decorator_deprecated.py:74]: cc @marimeireles and @SylvainCorlay

**Pull request merged**